src/olympia/files/utils.py

```diff
@@ -805,6 +806,45 @@ def update_version_number(file_obj, new_version_number):
     shutil.move(updated, file_obj.file_path)
 
 
+def write_crx_file(chunks, storage, loc):
```

Comment:
This could perhaps be better named `write_crx_as_xpi` or `hash_from_writing_crx` if you like...